src/components/CardChip/cardChip.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './cardChip.css';
+
+const CardChip = () => {
+  return (
+    <div className={'card-chip'} />
```

Comment:
```suggestion
    <div className='card-chip' />
```
문자열로만 이루어진 prop은 중괄호로 감쌀 필요가 없습니다